kunstmaan/redirect-bundle/5.2/config/routes/kunstmaan_redirect.yaml

```diff
@@ -0,0 +1,6 @@
+KunstmaanRedirectBundle:
+    resource: "@KunstmaanRedirectBundle/Resources/config/routing.yml"
+    # If your site is not multi language, remove the "{_locale}/" part of prefix and requirements
+    prefix:   /{_locale}/
+    requirements:
+        _locale: "%requiredlocales%"
```

Comment:
Where is this parameter defined?

Reply:
This is a parameter defined in the admin-bundle, which is a dependency of all bundles using this parameter. See [KunstmaanAdminBundle::KunstmaanAdminExtension](https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/422c988b093f1d9016a669864cc8a2408ced5146/src/Kunstmaan/AdminBundle/DependencyInjection/KunstmaanAdminExtension.php#L208-L209)